pkg/graph/resolvers/system_intake_actions_test.go

```diff
@@ -176,7 +176,7 @@ func (s *ResolverSuite) TestSystemIntakeRequestEditsAction() {
 		)
 		s.NoError(err)
 		s.Equal(intake.ID, actionedIntake.ID)
-		allActions, err := s.testConfigs.Store.GetActionsByRequestID(ctx, actionedIntake.ID)
+		allActions, err := s.testConfigs.Store.GetActionsBySystemIntakeID(ctx, actionedIntake.ID)
```

Comment:
should we use the resolver here and in the other tests below?

Reply:
I honestly don't think it matters much. You could make an argument that using the store methods isolates the tests from other resolver logic/tests, but you could also argue that using the resolvers is closer to how the actual resolvers behave. I just updated the method names here for expediency, but if you have a strong preference I'm willing to change them.

Reply:
ahh my bad, didn't realize it was just the names. i brought this up only because it's in the resolver package, but all good! thanks for the context